modules/tinymce/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html),
 and is generated by [Changie](https://github.com/miniscruff/changie).
 
+## Unrleased
```

Comment:
Since we are using changie now you should use that command to generate changelog items.

- [ ] Remove the direct change to the changelog file and use changie instead.

Reply:
Done!